config.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": 1,
+    "name": "DACCS E2E docker image updater config",
+
+    "project": {
+        "name": "birdhouse-deploy",
+        "url": "https://github.com/bird-house/birdhouse-deploy",
+        "bump_file": "birdhouse/default.env"
+    },
+
+    "images": [
+        {
+            "id": "finch",
+            "url": "https://hub.docker.com/r/birdhouse/finch",
+            "dockerhub_repo_name": "birdhouse/finch",
+            "bump_tag": "FINCH_IMAGE",
+            "tag_filter": "^version-[0-9]+(\\.[0-9]+)*$"
```

Comment:
For later if we will make this updater generic, able to handle other file format in other repos, for each image, we'll need something like in `config.json`:
```
"search_regex": "FINCH_IMAGE=\"\$DOCKERHUB_REPO_NAME:\$TAG_FILTER\"",  # this one probably replace "bump_tag".
"replace_template_string": "FINCH_IMAGE=\"\$DOCKERHUB_REPO_NAME:\$NEW_TAG_VALUE\""
```
Then we can run this on the `docker-compose.yml` file if we need to, just use a different search_regex.

Same idea as the bumpversion tool that can handle find-replace in any file, example bumpversion config in our cookiecutter-birdhouse repo: https://github.com/bird-house/cookiecutter-birdhouse/blob/f157ea8419fd4e32a9e4d52e613c90c807190e7e/%7B%7Bcookiecutter.project_repo_name%7D%7D/setup.cfg#L9-L23